pages/guides/plaid/generate-processor-token.mdx

```diff
@@ -22,6 +22,8 @@ To get started, verify that your Plaid account is enabled for the Dwolla integra
 
 A high-level overview of Plaid Link can be referenced below, however reference the [Plaid documentation](https://plaid.com/docs/auth/partnerships/dwolla/) for more information on integrating with Plaid Link.
 
+A sample app is available to help you get started with Plaid Link and Dwolla as well as demonstrate using on-demand authorization. You can find the sample app [here](https://github.com/Dwolla/integration-examples/tree/main/packages/plaid-funding-source).
```

Comment:
```suggestion
A sample app is available to help you get started with Plaid Link and Dwolla as well as demonstrate using on-demand authorization. Find the [Dwolla + Plaid Integration example](https://github.com/Dwolla/integration-examples/tree/main/packages/plaid-funding-source) in our Github page. 
```

Reply:
Adding a suggestion for avoiding adding links to vague texts like `"here"`. [Here's a quick article on why to avoid using `"here"` for links](https://www.smashingmagazine.com/2012/06/links-should-never-say-click-here/#here-conceals-what-users-are-clicking).

Feel free to change up the suggested sentence!